pages/common/smartctl.md

```diff
@@ -0,0 +1,23 @@
+# smartctl
+
+> View a disk's [SMART](https://en.wikipedia.org/wiki/S.M.A.R.T.) data and other information.
+
+- View SMART health summary:
+
+`sudo smartctl --health {{/dev/<device>}}`
```

Comment:
For filesystems, we usually give a concrete example- like `/dev/sda1`. It makes it more clear. Please do that for all examples.